### DISCUSSION TITLE
[New Futures] Two-Factor authentication

### DISCUSSION THREAD

**Discussion by piekny27** [piekny27/PulseBook#16] [Q&A] [answered]
<h3>Hi guys!</h3>
 I'd like to hear from you about our new future - Two-Factor Authentication. Please comment below! The comment with the most likes will be pinned as a reply

**Comment by adrianbeys** [comment, marked as answer]
I think it's a good idea to add authentication. To add such a module, you need to install packages.
<code class="language-plaintext highlighter-rouge">plivo==4.15.3</code>
<code class="language-plaintext highlighter-rouge">redis==3.5.3</code>
Clone the repository
```
$ git clone https://github.com/plivo-dev/2fa-python-demo
```
Replace the auth placeholders with your authentication credentials from the [Plivo console](https://console.plivo.com/dashboard/). Replace the phone number placeholder with an actual phone number. Replace the PHLO ID with null.

**Reply by piekny27** [reply]
Thanks bro!
